engintron.sh

```diff
@@ -313,6 +313,18 @@ EOFS
 		systemctl enable nginx
 	fi
 
+  if [ ! -f /etc/chkserv.d/engintron ]; then
```

Comment:
Not sure if this is relevant but I could only get this to work by using `nginx` instead of `engintron`.

ie `/etc/chkserv.d/nginx` and `nginx:1`